projects/habitat_ovmm/utils/config_utils.py

```diff
@@ -24,16 +24,16 @@ def get_habitat_config(
     return config, ""
 
 
-def get_ovmm_baseline_config(baseline_config_path: str) -> DictConfig:
+def get_omega_config(config_path: str) -> DictConfig:
     """Returns the baseline configuration."""
-    config = OmegaConf.load(baseline_config_path)
+    config = OmegaConf.load(config_path)
     OmegaConf.set_readonly(config, True)
     return config
 
 
 def merge_configs(
-    habitat_config: DictConfig, baseline_config: DictConfig
-) -> DictConfig:
+    habitat_config: DictConfig, baseline_config: DictConfig, env_config: DictConfig
```

Comment:
Merging (habitat, env) and ((habitat, env), baselines) configs should be decoupled. 

We should have one function that will return `env_config` and the other one that will merge `env_config` and `baseline_config`.

For example, if I want to create `HabitatOpenVocabManipEnv` I should have a way to get `env_config` without having `baseline_config`.